awx/ui_next/src/components/Lookup/Lookup.jsx

```diff
@@ -76,21 +80,33 @@ class Lookup extends React.Component {
     this.saveModal = this.saveModal.bind(this);
     this.getData = this.getData.bind(this);
     this.clearQSParams = this.clearQSParams.bind(this);
+    this.toggleDropdown = this.toggleDropdown.bind(this);
   }
 
   componentDidMount() {
-    this.getData();
+    Promise.all([this.getData()]);
```

Comment:
`Promise.all()` isn't doing anything here—for two reasons: 1. nothing awaits it/there's no `.then()` here and 2. `getData()` only returns a single promise, so it doesn't need to be run through `Promise.all()` to await it.